--- a/Makefile.am
+++ b/Makefile.am
@@ -33,8 +33,8 @@
 
 dist_pkgdata_SCRIPTS = uacme.sh
 
-if ENABLE_DOCS
 dist_man1_MANS = uacme.1
+if ENABLE_DOCS
 dist_html_DATA = uacme.1.html
 
 uacme.1: uacme.1.txt $(top_srcdir)/.version
--- a/Makefile.in
+++ b/Makefile.in
@@ -336,7 +336,7 @@
 	curlwrap.h json.c json.h jsmn.h msg.c msg.h $(am__append_1)
 BUILT_SOURCES = $(top_srcdir)/.version
 dist_pkgdata_SCRIPTS = uacme.sh
-@ENABLE_DOCS_TRUE@dist_man1_MANS = uacme.1
+dist_man1_MANS = uacme.1
 @ENABLE_DOCS_TRUE@dist_html_DATA = uacme.1.html
 EXTRA_DIST = GNUmakefile build-aux/git-version-gen uacme.sh uacme.1.txt \
 	     uacme.1 uacme.1.html
