--- a/Makefile.am
+++ b/Makefile.am
@@ -38,7 +38,7 @@
 dist_html_DATA = uacme.1.html
 
 uacme.1: uacme.1.txt $(top_srcdir)/.version
-	$(A2X) -d manpage -f manpage -a revision=$(VERSION) $<
+	$(A2X) -L -d manpage -f manpage -a revision=$(VERSION) $<
 
 uacme.1.html: uacme.1.txt $(top_srcdir)/.version
 	$(ASCIIDOC) -d manpage -b html5 -a revision=$(VERSION) $<
--- a/Makefile.in
+++ b/Makefile.in
@@ -927,7 +927,7 @@
 	echo $(VERSION) > $(distdir)/.tarball-version
 
 @ENABLE_DOCS_TRUE@uacme.1: uacme.1.txt $(top_srcdir)/.version
-@ENABLE_DOCS_TRUE@	$(A2X) -d manpage -f manpage -a revision=$(VERSION) $<
+@ENABLE_DOCS_TRUE@	$(A2X) -L -d manpage -f manpage -a revision=$(VERSION) $<
 
 @ENABLE_DOCS_TRUE@uacme.1.html: uacme.1.txt $(top_srcdir)/.version
 @ENABLE_DOCS_TRUE@	$(ASCIIDOC) -d manpage -b html5 -a revision=$(VERSION) $<
